Project1/style.css

```diff
@@ -1,13 +1,44 @@
 body {
-    background-color: #9cbabc;
+    background-color: #d2d3d3;
+
+}
+
+h1 {
+    font-family: 'Oswald', sans-serif;
+    font-size: 60px;
+    color: #b5afc3
+}
+
+h2 {
+    font-family: 'Oswald', sans-serif;
+    font-size: 40px;
+    color: #b5afc3
+}
+
+h3 {
+    font-family: 'Oswald', sans-serif;
+    font-size: 30px;
+    text-align: right;
 }
 
 p {
-    color: #000000;
+    display: grid;
+    grid-template-columns: auto auto auto auto;
+    font-family: 'Libre Franklin', sans-serif;
     font-size: 25px;
-    font-style: regular;
-    letter-spacing: 2px;
+    text-align: right;
     line-height: 1;
-    text-align: center;
-    font: Franklin Gothic Regular;
+
+
+}
+
+figcaption {
+    font-family: 'Libre Franklin', sans-serif;
+    font-size: 10px;
+    color: #8b8895
+}
+
+    {
```

Comment:
This ruleset is missing a selector -- what did you mean to apply these rules to? Possibly the `img` element?